src/styles.css

```diff
@@ -1,10 +1,52 @@
+/*
+Dark Green
+#0A210F;
+Medium Green
+#14591D;
+Light Green
+#99AA38;
+Yellow
+#E1E289;
+Blue
+#ACD2ED;
+*/
```

Comment:
Color palate selected to give the app a 'natural' feel, which should fit with our  highest probability users